02_cancer_type_classification/lasso_range_analysis/lasso_range_all.ipynb

```diff
@@ -17,6 +17,7 @@
    "source": [
```

Comment:
<p>Is it expected that theres a big difference in the number of genes in comparison to the previous run? (51480 now, 35520 before)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>The "shape" of this dataframe is the total number of models, i.e. one for each combination of gene/cancer type/LASSO parameter. In the most recent run we covered a larger range of LASSO parameters, so it makes sense that the dataframe is considerably larger.</p><p> </p><p>There are also two genes that are included now that weren't before, ALK and CIC. I slightly changed how the filtering for hypermutated samples works which results in these genes being included, so this is expected.</p>